components/badge.html

```diff
@@ -0,0 +1,146 @@
+            <div class="nav-li">
+              <a class="link" href="/components/card.html">
+                Card</a></div>
+            <div class="nav-li">
+              <a class="link" href="/components/image.html">
+                Image</a></div>
+            <div class="nav-li">
+              <a class="link" href="/components/input.html">
+                Input</a></div>
+            <div class="nav-li">
+              <a class="link" href="/components/list.html">
+                List</a></div>
+            <div class="nav-li">
+              <a class="link" href="/components/modal.html">
+                Modal</a></div>
+            <div class="nav-li">
+              <a class="link" href="/components/navigation.html">
+                Navigation</a></div>
+            <div class="nav-li">
+              <a class="link" href="/components/rating.html">
+                Rating</a></div>
+            <div class="nav-li">
+              <a class="link" href="/components/toast.html">
+                Toast</a></div>
+            <div class="nav-li">
+              <a class="link" href="/components/typography.html">
+                Typography</a></div>
+            
+          </ul>
+          </nav>
+
+
+        <main>
+       <div>
+         <div class="vl">
+         
+         <span>
+          <h1 class="main-heading-center">Badge</h1>
+          
+           
+          <div class="avatar-badge">
+          <img class="avatar-small" src="/img/img_avatar.png" />
+          <span class="badge-status online"></span>
+          </div>
+ 
+          <div class="avatar-badge">
+          <img class="avatar-small" src="/img/img_avatar.png" />
+          <span class="badge-status online1"></span>
+          </div>
+
+          <div class="iframe-container main-heading-center">
+            <iframe
+            src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=shades-of-purple&wt=none&l=htmlmixed&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522avatar-badge%2522%253E%250A%2520%2520%2520%253Cimg%2520class%253D%2522avatar-small%2520%2522%2520src%253D%2522%252Fimg%252Fimg_avatar.png%2522%2520%252F%253E%250A%2520%2520%2520%2520%253Cspan%2520class%253D%2522badge-status%2520online%2522%253E%253C%252Fspan%253E%250A%253C%252Fdiv%253E%250A%250A%253Cdiv%2520class%253D%2522avatar-badge%2522%253E%250A%2520%2520%2520%253Cimg%2520class%253D%2522avatar-small%2520%2522%2520src%253D%2522%252Fimg%252Fimg_avatar.png%2522%2520%252F%253E%250A%2520%2520%2520%2520%253Cspan%2520class%253D%2522badge-status%2520online1%2522%253E%253C%252Fspan%253E%250A%253C%252Fdiv%253E"
+            style="width: 639px; height: 348px; border:0; transform: scale(1); overflow:hidden;"
+            sandbox="allow-scripts allow-same-origin">
+          </iframe>
+         </div> 
+         <hr style="width:50%;text-align:center">
```

Comment:
avoid using inline style

Reply:
Okay , I will take care of it from now on.